BasicPlaybackSampleApp/BasicPlaybackSampleApp/Base.lproj/Main.storyboard

```diff
@@ -14,14 +15,14 @@
                         <color key="backgroundColor" white="1" alpha="1" colorSpace="calibratedWhite"/>
                         <prototypes>
                             <tableViewCell contentMode="scaleToFill" selectionStyle="blue" hidesAccessoryWhenEditing="NO" indentationLevel="1" indentationWidth="0.0" reuseIdentifier="Cell" textLabel="Arm-wq-HPj" style="IBUITableViewCellStyleDefault" id="WCw-Qf-5nD">
-                                <rect key="frame" x="0.0" y="86" width="320" height="44"/>
+                                <rect key="frame" x="0.0" y="86" width="600" height="44"/>
                                 <autoresizingMask key="autoresizingMask"/>
                                 <tableViewCellContentView key="contentView" opaque="NO" clipsSubviews="YES" multipleTouchEnabled="YES" contentMode="center" tableViewCell="WCw-Qf-5nD" id="37f-cq-3Eg">
-                                    <rect key="frame" x="0.0" y="0.0" width="320" height="43"/>
+                                    <rect key="frame" x="0.0" y="0.0" width="600" height="43.5"/>
```

Comment:
This is just storyboard.
On Nov 11, 2015 8:31 AM, "lental" <notifications@github.com> wrote:

> In
> BasicPlaybackSampleApp/BasicPlaybackSampleApp/Base.lproj/Main.storyboard
> <https://github.com/ooyala/ios-sample-apps/pull/52#discussion_r44552568>:
>
> >                                  <autoresizingMask key="autoresizingMask"/>
> >                                  <tableViewCellContentView key="contentView" opaque="NO" clipsSubviews="YES" multipleTouchEnabled="YES" contentMode="center" tableViewCell="WCw-Qf-5nD" id="37f-cq-3Eg">
> > -                                    <rect key="frame" x="0.0" y="0.0" width="320" height="43"/>
> > +                                    <rect key="frame" x="0.0" y="0.0" width="600" height="43.5"/>
>
> is this just an artifact of opening the BasicPlayback main.storyboard, or
> did you actually modify something?
>
> —
> Reply to this email directly or view it on GitHub
> <https://github.com/ooyala/ios-sample-apps/pull/52/files#r44552568>.
>